packages/rabbitmq/src/rabbitmq.interfaces.ts

```diff
@@ -116,6 +116,11 @@ export interface MessageHandlerOptions {
    * If set, will override the module's default deserializer.
    */
   deserializer?: MessageDeserializer;
+
+  /**
+   * Enables consumer-side batching.
+   */
+  batchOptions?: BatchOptions;
```

Comment:
For example, if you introduce a separate decorator where its tied to batching, this is no longer required here and its streamlined into the batching.

Now a decorator can have its own Symbol which will help you during the setup phase.
It may be ideal as we're always introducing optional properties